akka-actor-tests/src/test/java/akka/actor/ActorCreationTest.java

```diff
@@ -117,5 +141,25 @@ public void testRejectAbstractActorClass() {
       assertEquals(String.format("Actor class [%s] must not be abstract", H.class.getName()), e.getMessage());
     }
   }
-  
+
+  private static Creator<UntypedActor> createAnonymousCreatorInStaticMethod() {
+    return new Creator<UntypedActor>() {
+      @Override
+      public UntypedActor create() throws Exception {
+        return null;
+      }
+    };
+  }
+
+  @Test
+  public void testWrongAnonymousClassCreatedInStaticMethodCreator() {
+    final Creator<UntypedActor> thisIsNotStatic = createAnonymousCreatorInStaticMethod();
```

Comment:
I don't know why but I understood the inversed case from the ticket somehow. Will fix.